### PR TITLE
Added mobile sdk code samples

### PR DESCRIPTION
**Pull request by monica-m-ps**
**Please read before you review**

For formatting, the following tools are approved by @deniss-kaibagarovs and @Saamyyy.
Follow the procedure if you want to test for yourself:

**iOS**
Swift: 
- Used Swift format. Installed locally: `brew install swift-format`. 
- Press `option`+`shift`+`F`.

Objective-c:
https://formatter.org/objc-formatter

**Android**
Java:
https://www.tutorialspoint.com/online_java_formatter.htm

Kotlin:
https://facebook.github.io/ktfmt/

For other code samples in `configurations` folder, used ChatGPT for formatting Shell and PowerShell.

### CODE REVIEW

File: mobile-sdk/configurations/python/billing-address-and-email.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        },
+        "customerEmail": {
+            "collectionRequired": True
+        }
+    }
+})
+headers = {
+    'content-type': "application/json",
+    'version': "2022-04-07",
+    'Authorization': "Basic sk_sandbox_c8oLGaI__msxsXbpBDpdtwJEz_eIhfQoKHmedqgZPCdBx59zpKZLSk8OPLT0cZolbeuYJSBvzDVVsYvtpo5RkQ"  # <-- Change to your secret key
```
Comment (deniss-kaibagarovs) [inline mobile-sdk/configurations/python/billing-address-and-email.py:28]: same here, remove sk_sandbox and put YOUR_API_KEY

File: mobile-sdk/configurations/powershell/show-information.ps1 [hunk 2]
```diff
@@ -0,0 +1,44 @@
+# The sandbox API key passed in '$publicSandboxKey' is public.
+# Don't submit any personally identifiable information in any requests made with this key.
+# Sign in to developer.dojo.tech to create your own private sandbox key and use that instead
+# for secure testing.
+
+$publicSandboxKey = "sk_sandbox_c8oLGaI__msxsXbpBDpdtwJEz_eIhfQoKHmedqgZPCdBx59zpKZLSk8OPLT0cZolbeuYJSBvzDVVsYvtpo5RkQ"
```
Comment (deniss-kaibagarovs) [inline mobile-sdk/configurations/powershell/show-information.ps1:6]: sandbox key replace with YOUR_API_KEY

File: mobile-sdk/configurations/powershell/billing-address-and-email.ps1 [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# The sandbox API key passed in '$publicSandboxKey' is public.
+# Don't submit any personally identifiable information in any requests made with this key.
+# Sign in to developer.dojo.tech to create your own private sandbox key and use that instead
+# for secure testing.
+
+$publicSandboxKey = "sk_sandbox_c8oLGaI__msxsXbpBDpdtwJEz_eIhfQoKHmedqgZPCdBx59zpKZLSk8OPLT0cZolbeuYJSBvzDVVsYvtpo5RkQ"
```
Comment (deniss-kaibagarovs) [inline mobile-sdk/configurations/powershell/billing-address-and-email.ps1:6]: sk_sandbox

File: mobile-sdk/configurations/curl/billing-address-and-email.sh [hunk 4]
```diff
@@ -0,0 +1,18 @@
+# The sandbox API key passed in 'authorization' is public.
+# Don't submit any personally identifiable information in any requests made with this key.
+# Sign in to developer.dojo.tech to create your own private sandbox key and use that instead
+# for secure testing.
+
+curl -v --request POST \
+  --url https://api.dojo.tech/payment-intents \
+  --header 'content-type: application/json' \
+  --header 'Authorization: Basic sk_sandbox_kqCwRaWEsl9OM0xNSxh00eayT9k8hAqURGeNQseeV62rMxO5ZiZHub0-XRqkzPLfES2fVUl0seMOyujCIiYaMTaDyiBPf25b7W3Gr3oE0qbpTeM4MRuUMgtDt_scAEfJdqYBlBB_gt_31Bw7FihTISp2WIXXGeTD5WbwhdIhraIJVWR7jo2Hy-xKlBpK-lE8' \
```
Comment (deniss-kaibagarovs) [inline mobile-sdk/configurations/curl/billing-address-and-email.sh:9]: sk_sandbox

File: mobile-sdk/configurations/cs/show-information.cs [hunk 5]
```diff
@@ -0,0 +1,29 @@
+// The sandbox API key passed in 'ApiKeyClientAuthorization' is public.
+// Don't submit any personally identifiable information in any requests made with this key.
+// Sign in to developer.dojo.tech to create your own private sandbox key and use that instead
+// for secure testing.
+
+var paymentIntentsClient = new PaymentIntentsClient(new HttpClient(), new ApiKeyClientAuthorization("sk_sandbox_c8oLGaI__msxsXbpBDpdtwJEz_eIhfQoKHmedqgZPCdBx59zpKZLSk8OPLT0cZolbeuYJSBvzDVVsYvtpo5RkQ"));
```
Comment (deniss-kaibagarovs) [inline mobile-sdk/configurations/cs/show-information.cs:6]: sk_sandbox

File: mobile-sdk/configurations/cs/billing-address-and-email.cs [hunk 6]
```diff
@@ -0,0 +1,21 @@
+// The sandbox API key passed in 'ApiKeyClientAuthorization' is public.
+// Don't submit any personally identifiable information in any requests made with this key.
+// Sign in to developer.dojo.tech to create your own private sandbox key and use that instead
+// for secure testing.
+
+var paymentIntentsClient = new PaymentIntentsClient(new HttpClient(), new ApiKeyClientAuthorization("sk_sandbox_c8oLGaI__msxsXbpBDpdtwJEz_eIhfQoKHmedqgZPCdBx59zpKZLSk8OPLT0cZolbeuYJSBvzDVVsYvtpo5RkQ"));
```
Comment (deniss-kaibagarovs) [inline mobile-sdk/configurations/cs/billing-address-and-email.cs:6]: sk_sandbox

File: mobile-sdk/android/ui-customization/java/ui-customization.java [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+            "#DD000000",
+            "#DD000000",
+            "#DD000000",
+            "#DD000000",
+            "#DD000000",
+            "#DD000000");
+
+    // you can override any value for the darkColorPalette
+    DarkColorPalette darkColorPalette = new DarkColorPalette(
+            "#DD000000",
```
Comment (deniss-kaibagarovs) [inline mobile-sdk/android/ui-customization/java/ui-customization.java:28]: ask @Saamyyy to give you the updated version of this with // comments for each property

File: mobile-sdk/android/ui-customization/java/ui-customization.java [hunk 8]
```diff
@@ -0,0 +1,58 @@
+  // call this before u display the ui sdk on the screen
+  private void configureDojoThemeSettings(){
+    // you can override any value you for lightColorPalette
+    LightColorPalette lightColorPalette = new  LightColorPalette(
+            "#DD000000",
```
Comment (deniss-kaibagarovs) [inline mobile-sdk/android/ui-customization/java/ui-customization.java:5]: same here

File: mobile-sdk/android/payment-intent/kotlin/capture-google-pay.kts [hunk 9]
```diff
@@ -0,0 +1,14 @@
+// get the PaymentHandler
+val dojoPayUI = DojoSDKDropInUI.createUIPaymentHandler(this) { result -> showResult(result) }
+// on pay clicked
+dojoPayUI.startPaymentFlow(
+  DojoPaymentFlowParams(
+      paymentId= "pi_sandbox_RBMHTJ4fIkmSppDILZVCGw", //switch to production key when going live.
```
Comment (deniss-kaibagarovs) [inline mobile-sdk/android/payment-intent/kotlin/capture-google-pay.kts:6]: replace pi_sand.. with payment-intent-id

File: mobile-sdk/android/payment-intent/kotlin/capture-card.kts [hunk 10]
```diff
@@ -0,0 +1,18 @@
+// get the PaymentHandler
+val dojoPayUI = DojoSDKDropInUI.createUIPaymentHandler(this) { result->
+    // handle result
+    Toast.makeText(this, result.name, Toast.LENGTH_SHORT).show()
+}
+
+// on pay clicked
+dojoPayUI.startPaymentFlow(
+DojoPaymentFlowParams(
+  paymentId= "pi_sandbox_RBMHTJ4fIkmSppDILZVCGw",
```
Comment (deniss-kaibagarovs) [inline mobile-sdk/android/payment-intent/kotlin/capture-card.kts:10]: replace pi_sandbox.. with payment-intent-id

File: mobile-sdk/android/payment-intent/java/capture-google-pay.java [hunk 11]
```diff
@@ -0,0 +1,26 @@
...
+    false,
+    "merchantName",
+    "merchantId",
+    "gatewayMerchantId",
+    new ArrayList<>()
+);
+// on Pay tapped
+dojoPaymentFlowHandler.startPaymentFlow(
+    new DojoPaymentFlowParams(
+            "pi_sandbox_RBMHTJ4fIkmSppDILZVCGw", //switch to production key when going live.
```
Comment (deniss-kaibagarovs) [inline mobile-sdk/android/payment-intent/java/capture-google-pay.java:23]: replace pi_sandbox.. with payment-intent-id

File: mobile-sdk/android/payment-intent/java/capture-card.java [hunk 12]
```diff
@@ -0,0 +1,14 @@
...
+private final DojoPaymentFlowHandler dojoPaymentFlowHandler = DojoSDKDropInUI.INSTANCE.createUIPaymentHandler(
+          this,
+          ((dojoPaymentResult) -> {
+              return Unit.INSTANCE;
+          }));
+
+// on Pay tapped
+dojoPaymentFlowHandler.startPaymentFlow(
+        new DojoPaymentFlowParams(
+                "pi_sandbox_RBMHTJ4fIkmSppDILZVCGw",
```
Comment (deniss-kaibagarovs) [inline mobile-sdk/android/payment-intent/java/capture-card.java:11]: replace pi_sandbox.. with payment-intent-id

File: mobile-sdk/android/customer-secret/kotlin/create-customer-secret.kts [hunk 13]
```diff
@@ -0,0 +1,9 @@
+// get the PaymentHandler
+val dojoPayUI = DojoSDKDropInUI.createUIPaymentHandler(this) { result -> showResult(result) }
+// on "Pay" tapped
+dojoPayUI.startPaymentFlow(
+  DojoPaymentFlowParams(
+      paymentId= "pi_sandbox_RBMHTJ4fIkmSppDILZVCGw",
```
Comment (deniss-kaibagarovs) [inline mobile-sdk/android/customer-secret/kotlin/create-customer-secret.kts:6]: replace pi_sandbox.. with payment-intent-id

File: mobile-sdk/android/customer-secret/java/create-customer-secret.java [hunk 14]
```diff
@@ -0,0 +1,14 @@
...
+private final DojoPaymentFlowHandler dojoPaymentFlowHandler = DojoSDKDropInUI.INSTANCE.createUIPaymentHandler(
+        this,
+        ((dojoPaymentResult) -> {
+            return Unit.INSTANCE;
+        }));
+
+// on "Pay" tapped
+dojoPaymentFlowHandler.startPaymentFlow(
+      new DojoPaymentFlowParams(
+              "pi_sandbox_RBMHTJ4fIkmSppDILZVCGw",
```
Comment (deniss-kaibagarovs) [inline mobile-sdk/android/customer-secret/java/create-customer-secret.java:11]: replace pi_sandbox.. with payment-intent-id

File: manage-payments/objectivec/captureapplepay.m [hunk 15]
```diff
@@ -0,0 +1,11 @@
+#import dojo_ios_sdk
+
+DojoApplePayConfig *applePayConfig = [[DojoApplePayConfig alloc] initWithMerchantIdentifier: @"merchant.uk.co.paymentsense.sdk.demo.app"
+                                                                        collectBillingAddress: FALSE
+                                                                        collectShippingAddress: FALSE
+                                                                        collectEmail: FALSE];
+DojoApplePayPayload *applePayPayload = [[DojoApplePayPayload alloc] initWithApplePayConfig: applePayConfig email: NULL metaData: NULL isSandbox: YES];
+DojoPaymentIntent *paymentIntent = [[DojoPaymentIntent alloc] initWithConnecteToken: @"Token from Payment Intent (connecteToken)" totalAmount: [[DojoPaymentIntentAmount alloc] initWithValue: 120 currencyCode:@"GBP"]];  / TODO - this values should be populated from payment intent
```
[the remark this answers is not in the document]
Comment (monica-m-ps) [reply manage-payments/objectivec/captureapplepay.m:8]: Please ignore the manage-payments folder as I have removed the samples from there already. Mobile-sdk is the updated one

File: mobile-sdk/configurations/python/show-information.py [hunk 16]
```diff
@@ -0,0 +1,53 @@
...
+                  "value": 2500,
+                  "currencyCode": "GBP"
+              }
+        }
+    ],
+})
+headers = {
+    'content-type': "application/json",
+    'version': "2022-04-07",
+    'Authorization': "Basic sk_sandbox_c8oLGaI__msxsXbpBDpdtwJEz_eIhfQoKHmedqgZPCdBx59zpKZLSk8OPLT0cZolbeuYJSBvzDVVsYvtpo5RkQ"  # <-- Change to your secret key
```
[the remark this answers is not in the document]
Comment (monica-m-ps) [reply mobile-sdk/configurations/python/show-information.py:46]: For now let's leave this to maintain consistency

**Review by deniss-kaibagarovs — APPROVED** [review]

**Review by aaronforinton-dojo — APPROVED** [review]

**Review by RomanKondratyev — APPROVED** [review]

**Pull request merged**